pallets/funding/src/tests/4_community.rs

```diff
@@ -151,6 +154,161 @@ mod round_flow {
+					USD_DECIMALS,
+					ForeignAssets::decimals(AcceptedFundingAsset::DOT.to_assethub_id()),
+				)
+				.unwrap()
+			});
+
+			let mut funding_assets_cycle =
+				vec![AcceptedFundingAsset::USDT, AcceptedFundingAsset::USDC, AcceptedFundingAsset::DOT]
+					.into_iter()
+					.cycle();
+
+			let mut total_fundings_ct = Vec::new();
+			let mut total_fundings_usd = Vec::new();
+			let mut total_fundings_plmc = Vec::new();
+
+			let mut decimal_test = |decimals: u8| {
+				let funding_asset = funding_assets_cycle.next().unwrap();
+				dbg!(&funding_asset);
+				let funding_asset_usd_price = match funding_asset {
+					AcceptedFundingAsset::USDT => usdt_price,
+					AcceptedFundingAsset::USDC => usdc_price,
+					AcceptedFundingAsset::DOT => dot_price,
+				};
+
+				dbg!(decimals);
+				let mut project_metadata = default_project_metadata.clone();
+				project_metadata.token_information.decimals = decimals;
+				project_metadata.minimum_price =
+					<TestRuntime as Config>::PriceProvider::calculate_decimals_aware_price(
+						original_price,
+						USD_DECIMALS,
+						decimals,
+					)
+					.unwrap();
+
+				dbg!(original_price);
+				dbg!(project_metadata.minimum_price);
+				project_metadata.total_allocation_size = 1_000_000 * 10u128.pow(decimals as u32);
+				project_metadata.mainnet_token_max_supply = project_metadata.total_allocation_size;
+				project_metadata.participation_currencies = bounded_vec!(funding_asset);
+
+				let issuer: AccountIdOf<TestRuntime> = (10_000 + inst.get_new_nonce()).try_into().unwrap();
+				let evaluations = inst.generate_successful_evaluations(
+					project_metadata.clone(),
+					default_evaluators(),
+					default_weights(),
+				);
+				let project_id =
+					inst.create_community_contributing_project(project_metadata.clone(), issuer, evaluations, vec![]);
+
+				let total_funding_ct = project_metadata.total_allocation_size;
+				let total_funding_usd = project_metadata.minimum_price.saturating_mul_int(total_funding_ct);
+				let total_funding_plmc = usable_plmc_price.reciprocal().unwrap().saturating_mul_int(total_funding_usd);
+				let total_funding_funding_asset =
+					funding_asset_usd_price.reciprocal().unwrap().saturating_mul_int(total_funding_usd);
+
+				total_fundings_ct.push(total_funding_ct);
+				total_fundings_usd.push(total_funding_usd);
+				total_fundings_plmc.push(total_funding_plmc);
+
+				// Every project should want to raise 10MM USD
+				assert_eq!(total_funding_usd, 10_000_000 * USD_UNIT);
+
+				// Every project should produce the same PLMC bond when having the full funding at multiplier 1.
+				assert_close_enough!(total_funding_plmc, 1_190_476 * PLMC, Perquintill::from_float(0.999));
+
+				// Every project should have a different amount of CTs to raise, depending on their decimals
+				assert_eq!(total_funding_ct, 1_000_000 * 10u128.pow(decimals as u32));
+
+				// A minimum bid goes through. This is a fixed USD value, but the extrinsic amount depends on CT decimals.
```

Comment:
Look at comments and change where different from the bidding test. I think this should be both contribute and maximum instead of min? As you are contributing the max total funding amount